cabal-install/Distribution/Client/Security/DNS.hs

```diff
@@ -0,0 +1,134 @@
+module Distribution.Client.Security.DNS
```

Comment:
@23Skidoo Well, @dcoutts asked me to implement it in `cabal-install` for the time being...